src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,13 @@
+// En src/redux/categories/categoriesSlice.js
+import { createSlice } from '@reduxjs/toolkit';
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState: 'Under construction',
+  reducers: {
+    checkStatus: (state) => state,
+  },
+});
```

Comment:
- [ ] @mohashyne great work on your reducers, however in the categories reducer, the initial state should check first for the string `'Under construction'` before returning the state, also the initialState should have an empty categories array.. This is a project requirement as shown below
```suggestion
  reducers: {
     checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    },
  },
});
```
![image](https://github.com/mohashyne/book_store/assets/61408860/adf839dc-0c40-4626-82e4-65f3daaad1cb)